Core.DomainModel/Organization/Organization.cs

```diff
@@ -198,27 +198,17 @@ public Maybe<UIModuleCustomization> GetUiModuleCustomization(string module)
         {
             if (root == null) throw new ArgumentNullException(nameof(root));
 
-            IExternalOrganizationalHierarchyUpdateStrategy strategy;
-            //Pre-validate
-            switch (origin)
-            {
-                case OrganizationUnitOrigin.STS_Organisation:
-                    if (StsOrganizationConnection?.Connected != true)
-                    {
-                        return new OperationError($"Not connected to {origin:G}. Please connect before performing an update", OperationFailure.Conflict);
-                    }
-                    strategy = StsOrganizationConnection.GetUpdateStrategy();
-                    break;
-                case OrganizationUnitOrigin.Kitos:
-                    return new OperationError("Kitos is not an external source", OperationFailure.BadInput);
-                default:
-                    throw new ArgumentOutOfRangeException();
-            }
+            return GetStsOrganizationConnection(origin)
```

Comment:
![image](https://user-images.githubusercontent.com/13116889/204125401-e4eb7c1f-ede5-49d9-b6d4-b08f25902173.png)

@a-naskret Remind me of this entry in our talk as it is the design thing we will talk about